### PR TITLE
Add Restaurant List

### PR DESCRIPTION
**Pull request by Poonam30**
## Description
Listing Name, Address and Rating
## ScreenShot
Screenshot 2020-05-28 at 19 17 44

### CODE REVIEW

File: src/components/ShortRating.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+<img src = {starImage} alt="star" style={{width:"15px", height:"15px"}} />
+<text>{props.rating}/5({props.user_ratings_total})</text>
+</div>
+
+</div>
+  );
+};
+
+export default ShortRating;
```
Comment (helloanil) [inline src/components/ShortRating.js:19]: In this file, lines are not indented. Can you fix that?

Comment (Poonam30) [reply src/components/ShortRating.js]: Hi, I Fixed it

File: src/App.js [hunk 2]
```diff
@@ -1,32 +1,16 @@
 import React from "react";
-import logo from "./logo.svg";
 import "./App.css";
 import { BrowserRouter, Switch, Route } from "react-router-dom";
-
-function App() {
+import Restaurant from "./pages/RestaurantFrontPage";
```
Comment (helloanil) [inline src/App.js:4]: Can you change the name of the Component to `RestaurantsListingPage`

```js
import RestaurantsListingPage from "./pages/RestaurantFrontPage";
```

File: src/App.js [hunk 3]
```diff
@@ -1,32 +1,16 @@
...
-            Learn React
-          </a>
-        </header>
-
         <Switch>
           <Route
             path="/"
             exact
-            component={() => "todo: insert component with list of restaurants"}
+            component={Restaurant}
```
Comment (helloanil) [inline src/App.js:13]: Don't forget to change it here as well 🙏

File: src/components/RestaurantNameList.js [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+const RestaurantNameList = props => {
+  return (
+    <div >
+      <div className="nameList">
+        <text-corner> {props.readyInMinutes}mint</text-corner>
+        <img
+          className="restaurant-image"
+          src={props.icon ? props.icon : errorImage}
+          alt="icon"
+        />{" "}
```
Comment (helloanil) [inline src/components/RestaurantNameList.js:17]: You can remove `{" "}`

File: src/components/RestaurantNameList.js [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+      </div>
+      <text > {props.name} </text>{" "}
```
Comment (helloanil) [inline src/components/RestaurantNameList.js:19]: You can remove `{" "}`

File: src/components/RestaurantNameList.js [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+      <div className="inlineStyle" style={{ marginLeft: "5px" }}>
+        <img src={starImage} alt="star" style={{ width: "15px", height: "15px" }} />
+        <text>{props.rating}/5({props.user_ratings_total})</text>
+      </div>
+      <div className="inlneStyle">
+        <text > {props.formatted_address} </text>{" "}
```
Comment (helloanil) [inline src/components/RestaurantNameList.js:25]: You can remove `{" "}`

File: src/components/nameListStyle.css [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+    position: relative;
+    display: inline-flex;
+    text-align: left;
+
+  }
+  /* text {
+    font-size: 15px;
+    text-align: left;
+    width: 120px;
+  } */
```
Comment (helloanil) [inline src/components/nameListStyle.css:37]: Leaving commented code is not good. It's always better to remove them 👍

File: src/pages/RestaurantFrontPage.js [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+  loadData = async () => {
+    const response = await fetch(
+      "https://cors-anywhere.herokuapp.com/https://test-places-response.s3.eu-west-3.amazonaws.com/response.json"
+    );
+
+    const data = await response.json();
+    this.setState({
+      restaurantNameList: data.results
+    });
+    console.log(data.results);
```
Comment (helloanil) [inline src/pages/RestaurantFrontPage.js:20]: Also, we should remove `console.log`s from the code, when we are done with testing our code. You can remove this now.

File: src/pages/RestaurantFrontPage.js [hunk 9]
```diff
@@ -0,0 +1,45 @@
...
+          <button onClick={this.loadData}>Search</button>
+        </div>
+        <div >
+          {restaurantNameList.map(itemObject => (
+            <div className="gridlist">
+              <RestaurantNameList {...itemObject} />
+            </div>
+          ))}
+        </div>
+        {/* end return */}
```
Comment (helloanil) [inline src/pages/RestaurantFrontPage.js:38]: You can remove this commented code.

File: src/components/RestaurantNameList.js (hunk 6 above)
Comment (Poonam30) [reply src/components/RestaurantNameList.js:25]: I remove these things, but it comes automatically while formatting.

File: src/components/nameListStyle.css (hunk 7 above)
Comment (Poonam30) [reply src/components/nameListStyle.css]: I Removed all comments, and change the name.

**Pull request closed without merging**